lighthouse-viewer/app/index.html

```diff
@@ -35,6 +35,11 @@ <h1 class="viewer-placeholder__heading">Lighthouse Report Viewer</h1>
       or <a href="https://web.dev/measure">web.dev/measure</a>.</div>
       <input type="url" class="viewer-placeholder__url js-gist-url"
              placeholder="Enter or paste Gist URL">
+
+      <br>
+      <b>DevTools users:</b> there is a <a href="https://github.com/GoogleChrome/lighthouse/issues/12841">known bug</a>
+      where "Open in Viewer" does not work from the DevTools Lighthouse panel. This will be fixed on or around Sep 21.
```

Comment:
```suggestion
      where "Open in Viewer" does not work from the DevTools Lighthouse panel. This will be fixed in Chrome 94.
```

There are a shocking number of late/never updaters, so seems good to tie to the actual fix :) Happy to add `to be released on or around Sept 21` or something if you'd like to keep that user friendly bit.